src/binary-reversal/index.js

```diff
@@ -3,6 +3,20 @@
  *
  *  * @param {string} value
  */
-function binaryReversal(value) {}
+function convertToReveredBinary(value) {
+    if (value <= 0) {
+        return "";
+    }
+    let remainder = (value % 2).toString();
+    let newVal = Number.parseInt(value / 2, 0);
```

Comment:
These can be `const` too :)

Reply:
Noted, will enforce the use of const in my codes